static/app/views/eventsV2/queryList.tsx

```diff
@@ -234,8 +234,8 @@ class QueryList extends React.Component<Props> {
         <QueryGrid>{this.renderQueries()}</QueryGrid>
         <PaginationRow
           pageLinks={pageLinks}
-          onCursor={(cursor: string, path: string, query: Query, direction: number) => {
-            const offset = Number(cursor.split(':')[1]);
+          onCursor={(cursor, path, query, direction) => {
+            const offset = Number(cursor?.split(':')?.[1]);
```

Comment:
will it be fine in the case of `NaN`?

Reply:
const offset = Number(cursor?.split(':')?.[1] ?? 0); 🤷‍♀️  for a follow-up

Reply:
Oh weird, I actually thought that's exactly what I commited. But looks like it got lost somewhere.